lib/stimulus_reflex/callbacks.rb

```diff
@@ -0,0 +1,46 @@
+module StimulusReflex
+  module Callbacks
+    def self.included(klass)
```

Comment:
Considering that we have Rails as a dependency, we may want to look at using `ActiveSupport::Concern` and benefit from the prettier semantics. Thoughts? https://api.rubyonrails.org/classes/ActiveSupport/Concern.html

Reply:
Will happily rewrite if this is the direction 👍🏻

Reply:
made the change 👍